packages/scope/package.json

```diff
@@ -1,22 +1,28 @@
-{
-  "name": "@unocss/scope",
-  "type": "module",
-  "version": "0.59.0-beta.1",
-  "description": "Placeholder for UnoCSS scope import",
-  "author": "Anthony Fu <anthonyfu117@hotmail.com>",
-  "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/unocss/unocss",
-    "directory": "packages/scope"
-  },
-  "exports": {
-    ".": {
-      "import": "./index.mjs",
-      "require": "./index.cjs"
-    }
-  },
-  "main": "./index.mjs",
-  "module": "./index.mjs",
-  "types": "./index.d.ts"
+{
+  "name": "@unocss/scope",
+  "type": "module",
+  "version": "0.59.0-beta.1",
+  "description": "Placeholder for UnoCSS scope import",
+  "author": "Anthony Fu <anthonyfu117@hotmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/unocss/unocss",
+    "directory": "packages/scope"
+  },
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./index.d.ts",

```

Comment:
shoud be d.mts

Reply:
upps, there is no d.mts file...

Reply:
> from my review this should be ESM first, still missing node16-esm and bundler in your local? can you upload an screenshot?

These two PRs I created via github.com...

Reply:
ok, I'll try to review it later on my local

Reply:
![imagen](https://github.com/unocss/unocss/assets/6311119/91acc9c5-7c2d-4dc9-9ed2-3ba41445521f)